account_mass_reconcile_ref_deep_search/static/description/index.html

```diff
@@ -0,0 +1,422 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
you can not split in two file .html + one .ccs ?

Reply:
This is generated by oca-gen-readme :wink:

Reply:
ok thanks for explanation